### PR TITLE
Add article for BINAT with IPSEC like in core/#440

### PR DESCRIPTION
**Pull request by mimugmail**

### CODE REVIEW

File: source/manual/how-tos/ipsec-s2s-binat.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+===============
+IPSec BINAT
+===============
+
+Let's say company A has local LAN 10.0.1.0/24 and company B has local LAN 10.0.2.0/24.
```
Comment (fichtner) [inline source/manual/how-tos/ipsec-s2s-binat.rst]: Let's say -> Assume

File: source/manual/how-tos/ipsec-s2s-binat.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+We have to define new networks for the Phase 2 with unused ones and create NAT entries to reach the final systems.
+
+To make it easier we create a Phase with company A using 192.168.1.0/24 and company B using 192.168.2.0/24.
+Now we need to add on each side the local LAN in the field "Manual SPD entries". So for company A we set 10.0.1.0/24 in the field and for B 10.0.2.0/24.
+This allows the NAT process to speak with the Security Policy Database.
+
+Finally we have to create NAT entries since a client in LAN A (10.0.1.10) tries to reach 192.168.2.10, but this address has to be rewritten to 10.0.2.10 on Firewall B.
+
+Create the rule like in the screenshot and vice versa on Firewall A and you are good!
```
Comment (fichtner) [inline source/manual/how-tos/ipsec-s2s-binat.rst]: streiche " and you are good!"

setze "." oder ":"

**Comment by mimugmail** [comment]
done :)

2018-02-28 16:25 GMT+01:00 Franco Fichtner <notifications@github.com>:

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**